examples/with-okta-multi-tenant-pkce-flow/src/routes/dashboard.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {useState} from 'react';
+import { signOut } from "supertokens-auth-react/recipe/thirdparty";
+import Session from "supertokens-auth-react/recipe/session";
+import axios from 'axios'
+
```

Comment:
Add a comment here explaining that this is only accessible when the user is logged in

Reply:
done